fastly/alerts_test.go

```diff
@@ -23,14 +23,25 @@ func TestClient_FastlyAlerts(t *testing.T) {
 		Dimensions:         testDimensions,
 		EvaluationStrategy: testEvaluationStrategy,
 		IntegrationIDs:     []string{},
-		Metric:             ToPointer("status_5xx"),
+		Metric:             ToPointer("status_5xx_rate"),
 		Name:               ToPointer("test name"),
 		ServiceID:          ToPointer(TestDeliveryServiceID),
 		Source:             ToPointer("domains"),
```

Comment:
The metrics come from the `Domain Inspector` product. Therefore, it needs to be enabled!